sdk/src/main/assets/hcaptcha-form.html

```diff
@@ -101,13 +101,12 @@
             callback: function callback(token) {
                 return BridgeObject.onPass(token);
             },
-            'expired-callback': function expiredCallback() {
-                return BridgeObject.onError(15);
-            },
             'chalexpired-callback': function chalexpiredCallback() {
+                BridgeObject.onChallengeExpired();
                 return BridgeObject.onError(15);
```

Comment:
`BridgeObject.onChallengeExpired();` is equivalent to the below `bridgeObject.onError(15);`. Seems redundant thus `BridgeObject.onChallengeExpired()` can be removed.

Reply:
Fixed